frontend/src/pages/Sessions/SessionsFeedV3/SessionQueryBuilder/components/SessionFeedConfigurationV2/SessionFeedConfigurationV2.tsx

```diff
@@ -140,7 +140,7 @@ export const DropdownMenu = function ({
 						>
 							Order
 						</Text>
-						<Menu gutter={8}>
+						<Menu>
```

Comment:
the query builder input seems to be always focused as the input (for the key and value)

![image](https://github.com/highlight/highlight/assets/1351531/4c000030-5f42-4214-be33-6b4782db7589)

Reply:
Yeah, some interesting behavior changes with these components after the upgrade... Looks like @mayberryzane is refactoring a lot of this in https://github.com/highlight/highlight/pull/6442 as well, so maybe I'll wait to merge that in to resolve this.

Reply:
yup, sorry for the false alarm! looks like some CSS is outdated in reflame - i don't see that blue selection on the render.com preview.